llvm/test/CodeGen/AMDGPU/fix-illegal-copy.ll

```diff
@@ -0,0 +1,21 @@
+; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -mtriple=amdgcn -mcpu=gfx1100 -verify-machineinstrs -stop-after=finalize-isel | FileCheck %s -check-prefixes=GFX11
+
+define amdgpu_ps i32 @s_copysign_f32_bf16(float inreg %mag, bfloat inreg %sign.bf16) {
```

Comment:
This is a test to put somewhere else. The test here doesn't need anything more than any instruction that requires a VALU instruction. Shouldn't use bfloat